Gruntfile.js

```diff
@@ -85,10 +86,19 @@ module.exports = function(grunt) {
                     open: true
                 }
             }
+        },
+        'file-creator': {
```

Comment:
I don't really see the point in creating another file manually. Can't you just add it to the `module.js` and that way it'll be included in all the appropriate files.